samples/wifi/shell/overlay-zperf.conf

```diff
@@ -1,12 +1,12 @@
 CONFIG_NET_ZPERF=y
 CONFIG_POSIX_MAX_FDS=16
 # Zperf TCP=6, UDP=2, Others=1
-CONFIG_NET_SOCKETS_POLL_MAX=9
+CONFIG_NET_SOCKETS_POLL_MAX=16
 # Optimized networking settings for performance
 CONFIG_NET_PKT_RX_COUNT=30
-CONFIG_NET_PKT_TX_COUNT=30
+CONFIG_NET_PKT_TX_COUNT=29
 CONFIG_NET_BUF_RX_COUNT=30
-CONFIG_NET_BUF_TX_COUNT=60
+CONFIG_NET_BUF_TX_COUNT=58
 CONFIG_HEAP_MEM_POOL_SIZE=230000
```

Comment:
Does this change impact the peak TX throughput?

Reply:
IMHO it shouldn't, I have seen TX buffers having minimal impact, but please verify.